packages/video_player/video_player/android/src/main/java/io/flutter/plugins/videoplayer/Messages.java

```diff
@@ -297,25 +297,37 @@ static PositionMessage fromMap(HashMap map) {
   /** Generated class from Pigeon that represents data sent in messages. */
   public static class MixWithOthersMessage {
     private Boolean mixWithOthers;
+    private Boolean ambient;
```

Comment:
All of the Android changes can be reverted now; video_player was recently fully federated, and each platform has their own internal Pigeon-based channels.